src/Matterhorn/Config.hs

```diff
@@ -338,8 +338,10 @@ notifyVersion t =
     case t of
         "1" -> Right NotifyV1
         "2" -> Right NotifyV2
+        "3" -> Right NotifyV3
```

Comment:
Thanks also for adding this missing line. In the future, please include fixes like this in their own commits separately from commits that add new features. That way, if I needed to revert the feature commit due to a mistake, I would be able to leave these fixes in place.